content/Watsonx/Assistant/103.md

```diff
@@ -16,12 +16,14 @@ Learn about the artificial intelligence that runs out-of-the-box in watsonx Assi
 
```

Comment:
Also, fix the end of the lab by congratulating user on completion of lab + providing link for the next lab,